src/runtime_src/core/common/api/xrt_ip.cpp

```diff
@@ -223,11 +227,17 @@ class ip_impl
     return count++;
   }
 
+  static constexpr uint32_t isr_reg_addr = 0x0c;
```

Comment:
Maybe add a comment here, about what determines this address.  This is some implicit contract between HLS and XRT?

Reply:
done